helix-term/src/application.rs

```diff
@@ -144,22 +155,60 @@ impl Application {
 
             use futures_util::StreamExt;
 
+            #[cfg(not(windows))]
             tokio::select! {
                 biased;
 
                 event = reader.next() => {
                     self.handle_terminal_events(event)
                 }
+                Some(signal) = self.signals.next() => {
+                    use helix_view::graphics::Rect;
+                    match signal {
+                        signal::SIGTSTP => {
+                            self.restore_term().unwrap();
+                            low_level::emulate_default_handler(signal::SIGTSTP).unwrap();
+                        }                        signal::SIGCONT => {
```

Comment:
Not sure why this is messed up like this but seemed like `rustfmt` issue. https://github.com/rust-lang/rustfmt/issues/4913